mapbox_tilesets/utils.py

```diff
@@ -103,3 +108,59 @@ def validate_geojson(feature):
     }
 
     return validate(instance=feature, schema=schema)
+
+
+def _convert_precision_to_zoom(precision):
+    """Converts precision to zoom level based on the minimum zoom"""
+    if precision == "10m":
+        return 6
+    elif precision == "1m":
+        return 11
+    elif precision == "30cm":
+        return 14
+    else:
+        return 17
+
+
+def _tile2lng(tile_x, zoom):
+    """Returns tile longitude"""
+    return ((tile_x / 2 ** zoom) * 360.0) - 180.0
+
+
+def _tile2lat(tile_y, zoom):
+    """Returns tile latitude"""
+    n = np.pi - 2 * np.pi * tile_y / 2 ** zoom
+    return (180.0 / np.pi) * np.arctan(0.5 * (np.exp(n) - np.exp(-n)))
+
+
+def _calculate_tile_area(tiles):
+    """Returns tile area in square kilometers"""
+    EARTH_RADIUS = 6371.0088
+
+    left = np.deg2rad(_tile2lng(tiles[:, 0], tiles[:, 2]))
+    top = np.deg2rad(_tile2lat(tiles[:, 1], tiles[:, 2]))
+    right = np.deg2rad(_tile2lng(tiles[:, 0] + 1, tiles[:, 2]))
+    bottom = np.deg2rad(_tile2lat(tiles[:, 1] + 1, tiles[:, 2]))
+    return (
+        (np.pi / np.deg2rad(180))
+        * EARTH_RADIUS ** 2
+        * np.abs(np.sin(top) - np.sin(bottom))
+        * np.abs(left - right)
+    )
+
+
+def calculate_tiles_area(features, precision):
+    """Calculates the area of tiles
+
+    Parameters
+    ----------
+    features: list
+        features from GeoJSON source and coordinates
+
+    Returns
+    -------
+        total area of all tiles in square kilometers
+    """
+    zoom = _convert_precision_to_zoom(precision)
+    tiles = burn(features, min(zoom, MAXZOOM_TILE_COVER))
```

Comment:
See above note about `MAXZOOM_TILE_COVER
```suggestion
    tiles = burn(features, zoom)
```